src/library/autodj/autodjprocessor.cpp

```diff
@@ -1343,6 +1358,7 @@ void AutoDJProcessor::calculateTransition(DeckAttributes* pFromDeck,
     }
 
     switch (m_transitionMode) {
+//    case TransitionMode::RadioFullIntroOutro:
     case TransitionMode::FullIntroOutro: {
```

Comment:
remove?

Reply:
Yes.  This was an artifact of an earlier experiment.  I thought I had removed all of those.